Dockerfile

```diff
@@ -23,8 +23,10 @@ RUN make build && \
 
 FROM registry.redhat.io/ubi8-minimal
 
-COPY --from=builder /opt/app-root/src/insights-results-smart-proxy .
-COPY --from=builder /opt/app-root/src/openapi.json /openapi/openapi.json
+COPY --from=builder /opt/app-root/src/insights-results-smart-proxy . \
+     --from=builder /opt/app-root/src/server/api/v1/openapi.json /openapi/v1/openapi.json \
+     --from=builder /opt/app-root/src/server/api/v2/openapi.json /openapi/v2/openapi.json \
```

Comment:
I don't think that it will in that way. You should use `COPY` for each new path

Reply:
yes, I was unsure of that, TYVM for pointing it out

Reply:
fixd, TY!